Dockerfile.dev

```diff
@@ -1,7 +1,3 @@
 FROM ghcr.io/runatlantis/atlantis:latest
 COPY atlantis /usr/local/bin/atlantis
```

Comment:
From the comments below, I don't think this image is needed anymore.

@jamengual